drracket/drracket/info.rkt

```diff
@@ -70,15 +72,17 @@
     (framework:syntax-color:scheme:string ,tol-wob-constants)
     (framework:syntax-color:scheme:text ,tol-wob-constants)
     (framework:syntax-color:scheme:other ,light-blue)
-    (framework:syntax-color:scheme:parenthesis ,dark-plum)
+    (framework:syntax-color:scheme:parenthesis ,sea-green)
     (framework:syntax-color:scheme:symbol ,light-blue)
```

Comment:
Are these color changes intentional? If so, should they perhaps be in a separate commit (with a message explaining what was wrong before)?

Reply:
These changes added color settings for the new entries `'drracket:syncheck:unused-identifier` and `'drracket:syncheck:document-identifier`. I changed some old entries so that matching identifiers and document identifiers (and parenthesis in white-on-black schemes) don't share the same color. Of course, keeping the original color also seems good to me.